android-sdk/defaults/main.yml

```diff
@@ -1,10 +1,6 @@
 ---
 
-project_name: jenkins
 
-rhmap_version: 4.5
-# or
-buildfarm_templates_dir: /path/to/buildfarm_templates_dir
 config_file_dir: /path/to/config_file_dir
```

Comment:
config_file_dir can also be removed here. We are no longer using it in any tasks.